pubsub/google/cloud/pubsub_v1/gapic/publisher_client.py

```diff
@@ -0,0 +1,680 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/google/googleapis/blob/master/google/pubsub/v1/pubsub.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google engineers.
+#
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+"""Accesses the google.pubsub.v1 Publisher API."""
+
+import functools
+import pkg_resources
+
+import google.api_core.gapic_v1.client_info
+import google.api_core.gapic_v1.config
+import google.api_core.gapic_v1.method
+import google.api_core.grpc_helpers
+import google.api_core.page_iterator
+import google.api_core.path_template
+
+from google.cloud.pubsub_v1.gapic import publisher_client_config
+from google.cloud.pubsub_v1.proto import pubsub_pb2
+from google.iam.v1 import iam_policy_pb2
+from google.iam.v1 import policy_pb2
+from google.protobuf import field_mask_pb2
+
+_GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution(
+    'google-cloud-pubsub', ).version
```

Comment:
unintentional `,`?

Reply:
Autogen snafu of some kind. I will fix when I do my personal pass at it this week, but letting it go for now. Most likely the end result will be a carriage return before the `)` since the package names are variable length.